tests/Stripe/FileCreationTest.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace Stripe;
+
+class FileCreationTest extends TestCase
```

Comment:
Sorry for the nit, but given that the separate test file breaks convention elsewhere, could we add a short comment here that just explains that these tests have been moved to a separate suite because the methods they invoke use a separate API hostname?